From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Wed, 7 May 2025 13:46:39 +0200
Subject: test: skip using kernel-install due to bug #1104848

Test 12 fails on Debian:

```
TEST: UEFI boot (ukify, kernel-install)   [STARTED]
Not removing /tmp/autopkgtest-lxc.ctfqn41q/downtmp/autopkgtest_tmp/dracut-test.kzFcFU/dracut.dNC7kmJ.
Using ukify via kernel-install to create UKI
cp: cannot stat '/tmp/autopkgtest-lxc.ctfqn41q/downtmp/autopkgtest_tmp/dracut.conf.d/test/*': No such file or directory
cp: cannot stat '/tmp/autopkgtest-lxc.ctfqn41q/downtmp/autopkgtest_tmp/dracut.conf.d/uki-virt/*': No such file or directory
Not adding version '6.12.25-amd64', because kernel image not found.
No kernels to install found.
[...]
```

Skip using kernel-install in test 12 and wait for the kernel-install
issue reported in https://bugs.debian.org/1104848 to be fixed.

Forwarded: not-needed
---
 test/TEST-12-UEFI/test.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/TEST-12-UEFI/test.sh b/test/TEST-12-UEFI/test.sh
index 37bc4c4..d636871 100755
--- a/test/TEST-12-UEFI/test.sh
+++ b/test/TEST-12-UEFI/test.sh
@@ -57,7 +57,9 @@ test_setup() {
     # This is the preferred way to build uki with dracut on a systenmd based system
     # Currently this only works in a few distributions and architectures, but it is here
     # for reference
-    if command -v systemd-detect-virt &> /dev/null && systemd-detect-virt -c &> /dev/null \
+
+    # Skip using kernel-install due to https://bugs.debian.org/1104848
+    if false && command -v systemd-detect-virt &> /dev/null && systemd-detect-virt -c &> /dev/null \
         && command -v kernel-install &> /dev/null \
         && command -v systemctl &> /dev/null \
         && command -v ukify &> /dev/null \
